### DISCUSSION TITLE
RaiSim Windows

### DISCUSSION THREAD

**Discussion by takaratruong** [raisimTech/raisimLib#298] [Q&A] [answered]
Hi, I am running into an error  [ ModuleNotFoundError: No module named 'raisimpy' ] when trying to run the examples. I believe that I have added the correct path to the environment variable. In my case it is: 

C:\Users\takara\OneDrive\Documents\GitHub\raisim_ws\bin\ 

I have confirmed that the "raisimpy.cp38-win_amd64" file is in this directory. 

Any ideas on what I could be doing wrong? 

Thank you

**Comment by jhwangbo** [comment, marked as answer]
Do the versions match? Your python version vs the lib version

**Reply by takaratruong** [reply]
That was indeed the problem. The python version that I used to install it was different than what I had set as the environment variable. Thank you!
